db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
```

Comment:
Can we make sure to use the red logo on white background per https://www.mondialrelay.fr/solutionspro/documentation-technique/kit-de-communication/

![CleanShot 2024-02-02 at 12 41 00@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/10f3ec05-b757-4a5a-8c56-8d7687989711)

If you're up for it, include a mondial-relay_symbol.svg variant of just the symbol:


![CleanShot 2024-02-02 at 12 42 04@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/3ec4888d-79f3-4f1d-898f-deba34551408)